src/Tribe/REST/V1/Endpoints/Base.php

```diff
@@ -22,6 +22,54 @@ public function __construct( Tribe__REST__Messages_Interface $messages ) {
 		$this->messages = $messages;
 	}
 
+	/**
+	 * Returns a swagger structured array for the `requestBody` field.
+	 *
+	 * @param string $contentType The Content-Type header.
+	 * @param array $args
+	 * @since TBD
+	 *
+	 * @return array
+	 */
```

Comment:
```suggestion
	/**
	 * Returns a swagger structured array for the `requestBody` field.
	 *
	 * @param string              $contentType The Content-Type header.
	 * @param array<string|mixed> $args The provided post args.
	 *
	 * @since TBD
	 *
	 * @return array<string|mixed> The array of arguments for the swagger `requestBody` field.
	 */
```
A bit more info?